src/Fabric/Fabric.Autorest/test/New-AzFabricCapacity.Recording.json

```diff
@@ -0,0 +1,280 @@
+{
+  "New-AzFabricCapacity+[NoContext]+CreateExpanded+$PUT+https://management.azure.com/subscriptions/066c6e9e-c9ae-453d-bd2c-dacb1194fc0d/resourceGroups/rg-AzPowerShell-Tests-Fabric-bscyzitawq/providers/Microsoft.Fabric/capacities/abdcmtlgjfqz?api-version=2023-11-01+1": {
```

Comment:
expected request is "New-AzFabricCapacity+[NoContext]+CreateExpanded+$PUT+[https://management.azure.com/subscriptions/066c6e9e-c9ae-453d-bd2c-dacb1194fc0d/resourceGroups/rg-AzPowerShell-Tests-Fabric-agusodiexz/providers/Microsoft.Fabric/capacities/ovtpwudqfskr?api-version=2023-11-01+1"